lms/djangoapps/course_home_api/progress/v1/serializers.py

```diff
@@ -1,9 +1,59 @@
 """
 Progress Tab Serializers
 """
-
 from rest_framework import serializers
 from lms.djangoapps.course_home_api.outline.v1.serializers import CourseBlockSerializer
+from rest_framework.reverse import reverse
+
+
+class CoursewareSummarySerializer(serializers.Serializer):
+    """
+    Serializer for the courseware summary blocks
+    """
+    blocks = serializers.SerializerMethodField()
+
+    def get_blocks(self, blocks):
+        return [
+            {
+                'display_name': chapter['display_name'],
+                'chapter_url': reverse(  # currently points to the LMS url not the MFE
+                    'jump_to',
+                    kwargs={'course_id': str(self.context['course_key']), 'location': str(chapter['url_name'])},
+                    request=self.context['request'],
+                ),
```

Comment:
Where is the `chapter_url` used? I feel like in most spots, we only ever use subsection URLs and that's what is displayed on the progress page. This also makes me wonder if we can improve the way we fetch the subsection URL to just use the same format used elsewhere rather than this whole `courseware_section` reverse where you need to pass in a chapter too.